### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by hikmet2323githubio** [community/community#45851] [GitHub Education]
### Select Topic Area

Question

### Body

Hi 
Student Developer Pack my page is not continues. I clicked continue but than page is frozen. Why? I don't understand. I am waiting 4 days but still doesn't work.

**Comment by jge162** [comment]
Have you gotten to the point of submitting your supporting image yet?

**Comment by hikmet2323githubio** [comment]
[image: Screen Shot 2023-01-30 at 8.03.12 PM.png]

I closed and reopened many times, the page remained like this.

**Reply by jge162** [reply]
Cannot see your image you can not send image in a email reply. You must log into GitHub and go to discussion section and then upload the image into chat box.
